checkov/common/bridgecrew/vulnerability_scanning/integrations/docker_image_scanning.py

```diff
@@ -30,8 +30,7 @@ async def report_results_async(
             docker_image_name=kwargs["docker_image_name"],
         )
 
-    # can be removed, if image scanning is also using report_results_asyncio
-    # make sure to remove it only after validating it isn't used from the platform
+    # can be removed after it is fully moved to the platform
```

Comment:
@itai1357 can we move it completely?

Reply:
@nimrodkor will be in other PR, and after implementing it in platform